airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -639,6 +645,7 @@ def __init__(
         request_id: str | None = None,
         delete_on_error: bool = True,
         use_if_exists: bool = True,
+        num_retries_if_resource_is_not_ready: int | None = None,
```

Comment:
Can we stick to the int type only, like ` num_retries_if_resource_is_not_ready: int = 0`?

Reply:
Thank you, I have updated it